app/src/main/res/values/strings.xml

```diff
@@ -55,6 +55,10 @@
     <string name="addUnknownRecipientAlreadyUsed">The email is already used</string>
     <string name="addUnknownRecipientInvalidEmail">The email is invalid</string>
     <string name="addUnknownRecipientTitle">Add a recipient</string>
+    <string name="aiPromptPlaceholder">Tell the writing assistant what you want to write. For example, \"a thank-you email for a welcome gift\".</string>
```

Comment:
Usually, we don't put `.` at the end of sentences.

Reply:
We actually do sometimes but here the ending dot is due to it being in the original specs. This is the little help message displayed to the user when he is about to write prompt, it could make sens. I'm not very adamant about it

Reply:
I'd say that I think it a bit strange to have a full stop in the hint, but at the same times, it's 2 sentences, and the first one already is ending by a full-stop, so it wouldn't make sense that the second one doesn't I guess

Reply:
Ok for me !